ArcGISRuntimeSDKQt_CppSamples/Maps/MinMaxScale/README.md

```diff
@@ -0,0 +1,22 @@
+# Min-Max Scale
+
+This sample demonstrates how to set the minimum and maximum scale for a map.
+
+
+![](screenshot.png)
+
+## How to use the sample
+Use the mousewheel to zoom in and out of the map. Notice that the scale of which you can do that is limited by the application.
```

Comment:
this can be done on a touch screen as well, so include something like "or pinch gesture"